client/src/main/scala/drt/client/components/TerminalPlanningComponent.scala

```diff
@@ -35,7 +35,7 @@ object TerminalForecastComponent {
 
   val component = ScalaComponent.builder[Props]("TerminalForecast")
     .renderP((scope, props) => {
-      val sortedDays = props.forecastPeriod.days.toList.sortBy(_._1)
+      val sortedDays = props.forecastPeriod.forecast.days.toList.sortBy(_._1)
```

Comment:
maybe use a case on the sortBy to be explicit about what's being used for it, ie what _._1 is